src/mono/mono/mini/dwarfwriter.c

```diff
@@ -1338,10 +1338,10 @@ token_handler (MonoDisHelper *dh, MonoMethod *method, guint32 token)
 		if (method->wrapper_type) {
 			cmethod = (MonoMethod *)data;
 		} else {
-			ERROR_DECL (error);
 			cmethod = mono_get_method_checked (m_class_get_image (method->klass), token, NULL, NULL, error);
 			if (!cmethod)
 				g_error ("Could not load method due to %s", mono_error_get_message (error)); /* FIXME don't swallow the error */
+			mono_error_cleanup (error);
```

Comment:
same as above; is the cleanup real or just to appease the compiler?

Reply:
same comment as above, error won't be reused after cleanup so no need to re-init. In this case g_error (depending on used logger implementation), might not abort and there is no assert on cmethod not being NULL, but maybe it is more clear to just use mono_error_assert_msg_ok in this case as well.